app/controllers/user_details_controller.rb

```diff
@@ -1,2 +1,42 @@
 class UserDetailsController < ApplicationController
+  def index
+    user = UserDetail.includes(:user).all
+    render json: user, adapter: :json_api, status: 200
+  end
+
+  # def show
+  #   user_detail = UserDetail.includes(:user).find(params[:id])
+  # end
```

Comment:
Remove these comments, It is not good to commit commented codes.

Reply:
Thank you for letting me know. I removed the commented code. 💪